adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -264,6 +267,8 @@ private void handleEndpointConnectionWithAuthentication(final MqttEndpoint endpo
         if (endpoint.auth() == null) {
             LOG.trace("no auth information in endpoint found");
             endpoint.reject(MqttConnectReturnCode.CONNECTION_REFUSED_BAD_USER_NAME_OR_PASSWORD);
+            LOG.info("connection request from client [{}] rejected [{}]",
```

Comment:
should also be debug level FMPOV
There is no reason why any admin or other person should be particularly interested in this event, is there? If a client developer wonders why his device is rejected all the time, he may open an issue.